src/subcategories/id.ts

```diff
@@ -1,7 +1,21 @@
 import { makeEnum } from '@transcend-io/type-utils';
 
+/** Information that can be used to identify an individual */
 export const IdSubCategory = makeEnum({
-  // TODO: https://transcend.height.app/T-14003 - add more subcategories
+  /** A username associated with an individual */
+  Username: 'USERNAME',
+  /** An individual's phone number */
+  PhoneNumber: 'PHONE_NUMBER',
```

Comment:
Is this different from the `PHONE` subcategory under `Contact`?

Reply:
no i will remove